docs/docs/features/low-power-states.mdx

```diff
@@ -0,0 +1,355 @@
+            , <&gpio1 9  (GPIO_ACTIVE_HIGH)>
+            ;
+        row-gpios
+            = <&gpio0 19 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+            , <&gpio0 4  (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+            , <&gpio0 2  (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+            ;
+    };
+};
+```
+
+If the wakeup-only approach is used, then the above addition is skipped.
+
+With that in place, you would decorate the kscan driver:
+
+```
+/{
+    side_band_behavior_triggers: side_band_behavior_triggers {
+        compatible = "zmk,kscan-sideband-behaviors";
+        wakeup-source;
+        kscan = <&kscan>;
+        soft_off {
+            column = <0>;
+            row = <2>;
+            bindings = <&hw_soft_off>;
+        };
+    };
+};
+```
+
+Critically, the `column` and `row` values would correspond to the location of the added entry.
+
+Here are the properties for the kscan sideband behaviors node:
+
+- The `compatible` property for the node must be `zmk,kscan-sideband-behaviors`.
+- The `kscan` property is a phandle to the inner kscan instance that will have press/release events intercepted.
+
+The child nodes allow setting up the behaviors to invoke directly for a certain row and column:
+
+- The `row` and `column` properties set the values to intercept and trigger the behavior for.
+- The `bindings` property references the behavior that should be triggered when the matching row and column event triggers.
+
+For the wakeup-only approach, all soft_off child nodes are omitted.
+
+Lastly, which is critical, you would update the `zmk,kscan` chosen value to point to the new kscan instance:
+
+```
+/{
+    chosen {
+        ...
+        zmk,kscan = &side_band_behavior_triggers;
+        ...
+    };
+};
+```
+
+#### Soft Off Waker
+
+We need to add another device which will be enabled only when the keyboard is going into soft off state, and will configure the previously declared GPIO key with the correct interrupt configuration to wake the device from soft off once it is pressed.
+
+```
+/ {
+    wakeup_scan: wakeup_scan {
+        compatible = "zmk,gpio-key-wakeup-trigger";
+        trigger = <&soft_off_key>;
+        wakeup-source;
+        extra-gpios = <&gpio0 12 GPIO_ACTIVE_HIGH>
+    };
+};
+```
+
+Here are the properties for the node:
+
+- The `compatible` property for the node must be `zmk,gpio-key-wakeup-trigger`.
+- The `trigger` property is a phandle to the GPIO key defined earlier.
+- The `wakeup-source` property signals to Zephyr this device should not be suspended during the shutdown procedure.
+- An optional `extra-gpios` property contains a list of GPIO pins (including the appropriate flags) to set active before going into power off, if needed to ensure the GPIO pin will trigger properly to wake the keyboard. This is only needed for matrix integrated combos. For those keyboards, the list should include the matrix output needs needed so the combo hardware is properly "driven" when the keyboard is off.
```

Comment:
As per our discussion on Discord, this is necessary with the "wakeup-only" too, right? This below part might be read as it applies only to the combination case:

> This is only needed for matrix integrated combos. For those keyboards, the list should include the matrix output needs needed so the combo hardware is properly "driven" when the keyboard is off.

Maybe this bullet point just needs to be simplified, given we are already in the "matrix integrated" tab?

Reply:
Simplified as part of reshuffling